.gitlab-ci-check-golang-lint.yml

```diff
@@ -46,8 +46,6 @@ test:govendor-check:
   needs: []
   variables:
     GOLANG_VERSION: "1.16"
-  except:
-    - /^(master|hosted|staging|production|[0-9]+\.[0-9]+\.x|^saas-[a-zA-Z0-9.]+)$/
```

Comment:
while possible for release branches, IMO no change should land in `hosted|staging|production|^saas-[a-zA-Z0-9.]+` that didn't go through the master pipeline first - so we could skip these runner minutes there

Reply:
Better safe than sorry? I mean, the pipe is literally 10 seconds

Reply:
🚀 in that case, sure - run it everywhere 🎉